docker-compose.yaml

```diff
@@ -14,9 +14,18 @@ services:
     environment:
       - SPARK_MODE=master
       - SPARK_MASTER_WEBUI_PORT=8090
+      - MAX_EXECUTORS=4
+      - POSTGRES_USER=hive
+      - POSTGRES_PASSWORD=hivepassword
+      - POSTGRES_DB=hive
+      - POSTGRES_URL=postgres:5432
```

Comment:
Does the master really need the hive creds? I get the workers needing them

Reply:
As I read it, Hive config needs to be present in master, workers as well as driver nodes. That's why I put it there.

Reply:
:+1: